### PR TITLE
Fix when configuring Authentication from UI 

### PR DESCRIPTION
**Pull request by lawrencegripper**
Minor one, the redirect url the setup auto generates is missing a colon but it tripped me up when setting up auth.

Repro:

Deploy the site. 
Configure auth.
Signin doesn't work see blank page. 
Rummage around for a while trying to work out whats wrong before noticing the missing colon :)

### CODE REVIEW

File: client/src/components/Setup/index.tsx [hunk 1]
```diff
@@ -114,7 +114,7 @@ export default class Setup extends React.Component<any, ISetupState> {
       protocol = 'http';
     }
 
-    return protocol + '//' + host + '/auth/openid/return';
+    return protocol + '://' + host + '/auth/openid/return';
```
Comment (morsh) [inline client/src/components/Setup/index.tsx]: @lawrencegripper - This code is not correct. 
protocol == 'http:' or 'https:'

Comment (lawrencegripper) [inline client/src/components/Setup/index.tsx]: The protocol var is set to 'https' without a colon here: https://github.com/lawrencegripper/ibex-dashboard/blob/3d71f1d27b35b526796b113b0cda9dae01b9c927/client/src/components/Setup/index.tsx#L109

Without the added colon the resulting url is "https//examplehost.com/auth/openid/return"

Comment (morsh) [inline client/src/components/Setup/index.tsx]: You're right!
Can you fix the https ==> https: instead? This way it would make better sense.

**Comment by lawrencegripper** [comment]
Hopefully all fixed up!

**Comment by morsh** [comment]
@lawrencegripper - Thanks! I'll merge this tomorrow and test it out :)

**Pull request merged**